Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -236,4 +257,30 @@ class ThumbnailCell: UICollectionViewCell {
                 }
             })
     }
+
+    /**
+     Updates the insets and padding of the cell based on the size of the container collection view
+
+     - parameter size: Size of the container collection view
+     */
+    func updateLayoutForCollectionViewSize(size: CGSize) {
+        let cellInsets = ThumbnailCellUX.insetsForCollectionViewSize(size)
+        let imageInsets = ThumbnailCellUX.imageInsetsForCollectionViewSize(size)
+
+        if cellInsets != self.cellInsets {
+            self.cellInsets = cellInsets
+            imageWrapper.snp_remakeConstraints { make in
+                make.edges.equalTo(self.contentView).inset(cellInsets)
+            }
+        }
+
+        if imageInsets != self.imageInsets {
+            imageView.snp_remakeConstraints { make in
+                make.top.equalTo(self.imageWrapper).inset(imageInsets.top)
+                make.left.right.equalTo(self.imageWrapper).inset(imageInsets.left)
+                make.right.equalTo(self.imageWrapper).inset(imageInsets.right)
```

Comment:
You have `make.left.right` right above this -- don't think you meant to have `right` twice.